cmd/kube-apiserver/app/aggregator.go

```diff
@@ -205,6 +205,7 @@ var apiVersionPriorities = map[schema.GroupVersion]priority{
 	{Group: "apps", Version: "v1beta1"}:                          {group: 17800, version: 1},
 	{Group: "apps", Version: "v1beta2"}:                          {group: 17800, version: 1},
 	{Group: "apps", Version: "v1"}:                               {group: 17800, version: 15},
+	{Group: "events.k8s.io", Version: "v1beta1"}:                 {group: 17750, version: 5},
```

Comment:
I think the values here decide if `kubectl get events` resolves to the core group or the new group, is it intended to return the core group one?

Reply:
Yes, for compatibility